sdk/sphere/Azure.ResourceManager.Sphere/api/Azure.ResourceManager.Sphere.netstandard2.0.cs

```diff
@@ -0,0 +1,611 @@
+        public static Azure.ResourceManager.Sphere.Models.ProofOfPossessionNonceResponse ProofOfPossessionNonceResponse(string certificate = null, Azure.ResourceManager.Sphere.Models.CertificateStatus? status = default(Azure.ResourceManager.Sphere.Models.CertificateStatus?), string subject = null, string thumbprint = null, System.DateTimeOffset? expiryUtc = default(System.DateTimeOffset?), System.DateTimeOffset? notBeforeUtc = default(System.DateTimeOffset?), Azure.ResourceManager.Sphere.Models.ProvisioningState? provisioningState = default(Azure.ResourceManager.Sphere.Models.ProvisioningState?)) { throw null; }
+        public static Azure.ResourceManager.Sphere.Models.SignedCapabilityImageResponse SignedCapabilityImageResponse(string image = null) { throw null; }
+    }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct CapabilityType : System.IEquatable<Azure.ResourceManager.Sphere.Models.CapabilityType>
+    {
+        private readonly object _dummy;
+        private readonly int _dummyPrimitive;
+        public CapabilityType(string value) { throw null; }
+        public static Azure.ResourceManager.Sphere.Models.CapabilityType ApplicationDevelopment { get { throw null; } }
+        public static Azure.ResourceManager.Sphere.Models.CapabilityType FieldServicing { get { throw null; } }
+        public bool Equals(Azure.ResourceManager.Sphere.Models.CapabilityType other) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override bool Equals(object obj) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override int GetHashCode() { throw null; }
+        public static bool operator ==(Azure.ResourceManager.Sphere.Models.CapabilityType left, Azure.ResourceManager.Sphere.Models.CapabilityType right) { throw null; }
+        public static implicit operator Azure.ResourceManager.Sphere.Models.CapabilityType (string value) { throw null; }
+        public static bool operator !=(Azure.ResourceManager.Sphere.Models.CapabilityType left, Azure.ResourceManager.Sphere.Models.CapabilityType right) { throw null; }
+        public override string ToString() { throw null; }
+    }
+    public partial class CatalogPatch
+    {
+        public CatalogPatch() { }
+        public System.Collections.Generic.IDictionary<string, string> Tags { get { throw null; } }
+    }
+    public partial class CertificateChainResponse
+    {
+        internal CertificateChainResponse() { }
+        public string CertificateChain { get { throw null; } }
+    }
+    public partial class CertificateProperties
```

Comment:
Could we rename this type to `SphereCertificateProperties`?

Reply:
fixed in latest commit